### PR TITLE
Fix compatibility with Python 3.5

### PR DESCRIPTION
**Pull request by duedal**
The enum34 package must not be installed on Python 3.5+, for more info see https://bitbucket.org/stoneleaf/enum34/issues/5

Also fix an issue with the unit tests, apparently the amount of whitespace in one of the tests has changed.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -12,8 +13,11 @@
 requires = [
     'requests>=1.0.0,<3.0',
     'pycountry',
-    'enum34',
 ]
+# Incompatible with Py 3.5+: https://bitbucket.org/stoneleaf/enum34/issues/5
+if (sys.version_info[0] < 3 or
+        sys.version_info[0] == 3 and sys.version_info[1] < 5):
+    requires.append('enum34')
```
Comment (keleshev) [inline setup.py]: If you're pedantic this should be `< 4` and not `< 5`. The package is called `enum34` because it back-ports enums from  Python 3.4, so you don't need to install it for 3.4 either.

Comment (keleshev) [inline setup.py]: An overall better way to write this would be:

```python
if sys.version_info < (3, 4):
    requires.append('enum34')
```

**Comment by martinleblanc** [comment]
Support for python 3.5. has been added with #19

**Comment by EvanPurkhiser** [comment]
enum34 was never fixed in master, and to this day it seems this still installs enum34 in python > 3.3
